### PR TITLE
Button note

### PR DESCRIPTION
**Pull request by ColinCollins**
Re: https://github.com/cocos-creator/2d-tasks/issues/1060

1. 修改 button 注释
2. 增添监听事件注释

### CODE REVIEW

File: cocos2d/core/components/CCButton.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ const State = cc.Enum({
  *  Button.EVENT_HOVER_IN<br/>
  *  Button.EVENT_HOVER_MOVE<br/>
  *  Button.EVENT_HOVER_OUT<br/>
+ * The user can get the current clicked object through the 'event.target' which is from the clickEvent callback function.
```
Comment (pandamicro) [inline cocos2d/core/components/CCButton.js]: User can get the current clicked node with 'event.target' from event object which is passed as parameter in the callback function of click event.

File: cocos2d/core/components/CCButton.js [hunk 2]
```diff
@@ -105,7 +106,7 @@ const State = cc.Enum({
  *   -cc.Node.EventType.MOUSE_LEAVE // 鼠标离开目标事件<br/>
  *   -cc.Node.EventType.MOUSE_UP    // 鼠标松开事件<br/>
  *   -cc.Node.EventType.MOUSE_WHEEL // 鼠标滚轮事件<br/>
- *
+ * 用户可以通过获取 __点击事件__ 回调参数属性 event.target 获取当前点击对象。
```
Comment (pandamicro) [inline cocos2d/core/components/CCButton.js]: 用户可以通过获取 __点击事件__ 回调函数的参数 event 的 target 属性 event.target 获取当前点击对象。

File: cocos2d/core/CCNode.js [hunk 3]
```diff
@@ -3094,29 +3094,57 @@ var Node = cc.Class({
 });
 
 /**
+ * !#en
+ * The position changing event, you can listen this event through the statement cc.node.on(cc.Node.EventType.POSITION_CHANGED, callback, this);
```
Comment (pandamicro) [inline cocos2d/core/CCNode.js]: The position changing event, you can listen this event through the statement 

->

The position changing event, you can listen to this event through the statement

**Review by pandamicro — CHANGES_REQUESTED** [review]

**Review by pandamicro — APPROVED** [review]

File: cocos2d/core/CCNode.js [hunk 4]
```diff
@@ -3094,29 +3094,57 @@ var Node = cc.Class({
...
+ * !#zh
+ * 删除子节点监听事件，通过 cc.node.on(cc.Node.EventType.CHILD_REMOVED, callback, this); 进行监听。
  * @event child-removed
  * @param {Node} child - child which have been removed
  */
 /**
+ * !#en
+ * The reordering child event, you can listen to this event through the statement cc.node.on(cc.Node.EventType.CHILD_REORDER, callback, this);
+ * !#zh
+ * 自节点顺序变动监听事件，通过 cc.node.on(cc.Node.EventType.CHILD_REORDER, callback, this); 进行监听。
```
Comment (jareguo) [inline cocos2d/core/CCNode.js:3139]: 子节点顺序变动事件

Comment (jareguo) [reply cocos2d/core/CCNode.js:3139]: cc.node 是什么鬼？

**Pull request merged**